root/static/scripts/edit/URLCleanup.js

```diff
@@ -1929,7 +1929,26 @@ const CLEANUPS = {
     match: [new RegExp('^(https?://)?(www\\.)?mainlynorfolk\\.info', 'i')],
     type: LINK_TYPES.otherdatabases,
     clean: function (url) {
-      return url.replace(/^(?:https?:\/\/)?(?:www\.)?mainlynorfolk\.info\//, 'https://mainlynorfolk.info/');
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?mainlynorfolk\.info\//, 'https://mainlynorfolk.info/');
+      // Drop unneeded index.html from artist pages
+      url = url.replace(/^https:\/\/mainlynorfolk\.info\/([^/]+)\/index\.html$/, 'https://mainlynorfolk.info/$1/');
```

Comment:
We might want to add an ending slash `/` if it is missing:

```suggestion
      url = url.replace(/^https:\/\/mainlynorfolk\.info\/([^/]+)(?:\/index\.html)?$/, 'https://mainlynorfolk.info/$1/');
```